scripts/ice-road-pipeline.py

```diff
@@ -49,10 +52,22 @@
         log.setLevel(logging.DEBUG)
 
     # run main function
-    outtif, outlas = las2uncorrectedDEM(in_dir, debug, log)
+    if user_dem == 'False':
```

Comment:
Turns out if you don't give a default and the user doesn't use the flag then docopt will automatically make this variable False. So just remove [default: False] from the option in the top doc string and you can remove this.